docs/en/engines/table-engines/mergetree-family/replication.md

```diff
@@ -55,7 +55,7 @@ Example of setting the addresses of the ZooKeeper cluster:
 ```
 
 ClickHouse also supports to store replicas meta information in the auxiliary ZooKeeper cluster by providing ZooKeeper cluster name and path as engine arguments.
-In other word, it supports to store the metadata of differnt tables in different ZooKeeper clusters.
+In other word, it supports to store the metadata of different tables in different ZooKeeper clusters.
```

Comment:
```suggestion
In other word, it supports storing the metadata of different tables in different ZooKeeper clusters.
```
Does this sound better? `supports to store` sounds a bit weird to me, but I'm not sure.
I know it's unrelated but while we're here... 😄